### DISCUSSION TITLE
about tokenURI and nft

### DISCUSSION THREAD

**Discussion by shobit000** [Cyfrin/foundry-full-course-cu#2526] [Q&A]
can anyone explain me about tokenURI and
how the user sees a nft and interact with it

**Comment by EngrPips** [comment]
Hello @shobit000, tokenURI is simply the `Uniform Resource Identifier` for the token, which we can always use to Identify the token. So, with the URI, we get all the details of the token, including its image. I am not sure what you mean by `how the user sees an nft and interacts with it.`

**Reply by shobit000** [reply]
thanks for the info

**Reply by EngrPips** [reply]
You are welcome.
